ui/site/src/account.ts

```diff
@@ -1,10 +1,11 @@
 import * as xhr from 'common/xhr';
 
 lichess.load.then(() => {
-  const localPrefs: [string, string, string, number][] = [
+  const localPrefs: [string, string, string, number | string][] = [
```

Comment:
I see we've graduated from using numbers for booleans, to using numbers OR strings for booleans.

Reply:
I had hoped to use this capacity to start sending the strings 'zero', 'one', 'two', etc. to represent integers.  Why do you always have to ruin everything?